syntax = "proto3";

package rpc_model.echo;

// The greeting service definition.
service Api {
  // Sends a greeting
  rpc DoEcho (EchoRequest) returns (EchoReply) {}
}

// The request message containing the user's name.
message EchoRequest {
  string message = 1;
}

// The response message containing the greetings
message EchoReply {
  string message = 1;
}